GSoC-2025/Prisha_Sharma.md

```diff
@@ -0,0 +1,196 @@
+- **GitHub username**: parharti
+- **Zulip username**: Prisha Sharma
+- **Location & time-zone**: India, Indian Standard Time (GMT+5:30)
+- **Personal website / project portfolio**: https://github.com/parharti
+- **Code contribution**
+<blockquote>
+    <table>
+        <tr>
+            <td rowspan="5"><strong>PR</strong></td>
+            <td><strong>Cellfinder</strong></td>
+            <td><a href="https://github.com/brainglobe/cellfinder/pull/490">#490</a></td>
+            <td>Inform the user when they supply just the weights in place of the full model (merged)</td>
+        </tr>
+        <tr>
+            <td><strong>Cellfinder</strong></td>
+            <td><a href="https://github.com/brainglobe/cellfinder/pull/499">#499</a></td>
+            <td>Remove save weights option from training widget (merged)</td>
+        </tr>
+        <tr>
+            <td><strong>Datashuttle</strong></td>
+            <td><a href="https://github.com/neuroinformatics-unit/datashuttle/pull/470">#470</a></td>
+            <td>Attempting to copy in the terminal user interface crashes on headless HPC (merged)</td>
+        </tr>
+    </table>
+</blockquote>
+
+
+
+## Project proposal
+
+_Length: max 1 page_
+
+**Synopsis**
+
+This project aims to enhance CellFinder by adding multi-channel support for brain image classification. The goal is to extend CellFinder’s capabilities to process and classify cells from an arbitrary number of channels, making it more adaptable and improving its usability across different datasets. This enhancement is crucial, as some datasets may contain only a single signal channel, while others may have multiple signal channels carrying valuable information. By improving flexibility and accuracy, this project will make CellFinder a more powerful tool for the open-source neuroscience community.
+
+### Deliverables
+
+- Modify blob detection to support multiple channels.
+
+- Enhance neural network for multi-channel cell classification.
+
+- Ensure comprehensive testing for new functionalities.
+
+- Provide detailed documentation for ease of use.
+
+- Publish a blog post showcasing improvements with 1 & 3-channel images.
+
+-  Bonus (if time allows): Implement a visualization tool to display detected cells across different channels, aiding interpretation.
+
+**Implementation timeline**
+
+<table>
+  <thead>
+    <tr>
+      <th>Timeline</th>
+      <th>Deliverables</th>
+      <th>Hours/Week</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><b>Community Bonding Period</b><br>Focus: Understanding</td>
+      <td>
+        <ol type="1">
+          <li>Discuss project details with mentor and understand the codebase</li>
+          <li>Explore existing multi-channel imaging datasets.</li>
+          <li>Set up the development environment and resolve dependencies</li>
+        </ol>
+      </td>
+      <td>15</td>
+    </tr>
+    <tr>
+      <td><b>Week 1-2</b><br>Focus: Integrating attention into blob detection</td>
```

Comment:
The cell candidate detection uses classical filters and thresholding. This doesn't have (and shouldn't have) any ML component. Supporting multiple channels here should be pretty simple and may include:
- Simply allowing the user to pass N channels as input
- Running cell candidate detection on N channels and merging the results